docker/hipsycl/Dockerfile

```diff
@@ -15,7 +15,6 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
     rm -rf /var/lib/apt/lists*
 
 RUN git clone https://github.com/illuhad/hipSYCL.git \
-      --branch=develop --single-branch --shallow-since=2021-08-01 \
```

Comment:
I am curious about this `--shallow-since=2021-08-01`.
If you want to minimize transfers you could still use some shallowing with `--depth 1 --shallow-submodules --branch $IMPL_VERSION`